src/client.cpp

```diff
@@ -1068,6 +1069,13 @@ void CClient::ProcessAudioDataIntern ( CVector<int16_t>& vecsStereoSndCrd )
                               static_cast<float> ( iReverbLevel ) / AUD_REVERB_MAX / 4 );
     }
 
+    // apply a general gain to all audio input:
```

Comment:
Don't enter this block unless it's needed.  It's got a loop, so it's a fair saving for a simple `if`.

Reply:
Thanks, done.